### DISCUSSION TITLE
A repository always show in my notification repository list

### DISCUSSION THREAD

**Discussion by penglei** [community/community#38846] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I cleared all my notifications , but the `ajaxorg/ace` repository still stays in the left sidebar.

![image](https://user-images.githubusercontent.com/1140325/201295936-3e765d1d-f6ac-4297-9813-61b623d164a0.png)

**Comment by radar-blip** [comment]
You should be able to select it (click on it) and mark as done.

**Reply by penglei** [reply]
No `Done` button is presented 😞

**Reply by radar-blip** [reply]
It doesn't present itself like a vampire with magic powers. Its already there on your page.

![here](https://user-images.githubusercontent.com/39196767/201438064-204b7a4d-ecb9-496e-bb0a-b7a7d022452e.png)

**Comment by DSDmark** [comment]
Hi @penglei,

Maybe you are right. I have seen similar discussion about that :- 
(https://github.com/community/community/discussions/38174)

I don't know why this happening with some peoples hope it's will fix soon.

**Reply by DSDmark** [reply]
And also change your category into **issue**.

**Comment by Habeel06** [comment]
Did you try  marking it as done

**Reply by penglei** [reply]
Yes, I did.

**Comment by penglei** [comment]
My investigation yielded some details. I moved a message form Done category to Inbox again, it shows that there are still 4 messages, however, only one is displayed(The one I moved from Done list).  I think the message count information in the server is corrupted for my account. 
![71B554E5-2023-47B8-BEFA-5506E40FFD37](https://user-images.githubusercontent.com/1140325/201452809-60ca759b-2796-482e-a518-7481e49ce50a.jpeg)

**Comment by jfullstackdev** [comment, marked as answer]
@penglei if you are asking how to remove it, here is the answer, I have been using it and it's always working

https://github.com/orgs/community/discussions/6874#discussioncomment-2859125

**Reply by penglei** [reply]
Thanks , it didn't work for me. However, I found another effective solution in the discusstion, it worked.😃  Thanks, again!
https://github.com/orgs/community/discussions/6874#discussioncomment-1872442
